CheckDrive.Web/CheckDrive.Web/Models/Account.cs

```diff
@@ -0,0 +1,24 @@
+﻿namespace CheckDrive.Web.Models
+{
+    public class Account
+    {
+        public int Id { get; set; }
+        public string? Login { get; set; }
+        public string? Password { get; set; }
+        public string? PhoneNumber { get; set; }
+        public string? FirstName { get; set; }
+        public string? LastName { get; set; }
+        public DateTime Bithdate { get; set; }
+
+        public int RoleId { get; set; }
+        public Role? Role { get; set; }
+
+        public ICollection<Dispatcher>? Dispatchers { get; set; }
```

Comment:
In here should be virtual(public virtual ICollection)

Reply:
Thanks for the your feedback, but what will change if I use it? Can you explain if possible

Reply:
The virtual keyword enables polymorphism, allowing derived classes to override the property, while its absence indicates a non-overridable property.

Reply:
I know that. But I removed the virtual keyword from our old projects and didn't notice any change

Reply:
https://stackoverflow.com/questions/25715474/why-navigation-properties-are-virtual-by-default-in-ef